manual_testcases/login-validation.yaml

```
@@ -0,0 +1,67 @@
+---
+feature: Login field validation
+
+scenarios:
+  - scenario: Show error message when username field is empty
+    given:
+      - the user is on the login page
+    when:
+      - the user leaves the username field empty
+      - the user clicks the login button
```

Please follow provided schema: https://github.com/abdullah-aladham/automationStore-CypressTS/blob/master/manual_testcases/sample.yml

```suggestion
    when: the user leaves the username field empty
     and: the user clicks the login button
```

this is valid for the rest of the file